helm/charts/td-agent-barito/templates/daemonset.yaml

```diff
@@ -42,3 +42,18 @@ spec:
         - name: td-agent-barito-config
           configMap:
             name: {{ template "td-agent-barito.fullname" . }}
+
+      {{- with .Values.nodeSelector }}
```

Comment:
just want to make sure, if `nodeSelector` or `affinity` or `tolerations` are blank.. this key won't be generated, right?

Reply:
cc @akhy

Reply:
@giosakti yes,`with` keyword only generates the content when the value is not null/empty

confirmed by running `helm template helm/charts/td-agent-barito` (default values)